pkg/plugins/crypto/crypto.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// Package crypto provides the default implementation of pkg/framework/spi/crypto.Crypto interface
+//
+// It uses github.com/tink/go crypto primitives
+package crypto
+
+import (
+	"errors"
+
+	"github.com/google/tink/go/aead"
+	"github.com/google/tink/go/core/primitiveset"
+	"github.com/google/tink/go/keyset"
+	aeadsubtle "github.com/google/tink/go/subtle/aead"
+	"golang.org/x/crypto/chacha20poly1305"
+
+	cryptoapi "github.com/hyperledger/aries-framework-go/pkg/framework/spi/crypto"
+)
+
+// Crypto is the default Crypto implementation
+type Crypto struct {
+}
+
+// Assert that Crypto implements the Crypto interface.
+var _ cryptoapi.Crypto = (*Crypto)(nil)
```

Comment:
This assertion should not be in a code file. This is a test.

Reply:
Also odd how the linter missed this global variable. Please avoid globals :).

Reply:
this statement enforces the implementation to follow the interface functions and will provide a compilation error if it doesn't.. it's useful during refactoring.

the variable name is omitted here :) ... it's just the type that's defined to enforce the type checking, so there is no global variable defined in this line

Reply:
Please move to _test.go.

Our automated tests ensure that this has compliance. It should not be in the code.

Reply:
done